camel/model_backend.py

```diff
@@ -81,9 +108,24 @@ class ModelFactory:
     """
 
     @staticmethod
-    def create(model_type: ModelType, model_config_dict: Dict) -> ModelBackend:
+    def create(model_type: Optional[ModelType],
+               model_config_dict: Dict) -> ModelBackend:
```

Comment:
Why don't we set both of them as kwargs if we want default model_type to be `ModelType.GPT_3_5_TURBO`?
```suggestion
    def create(model_type: ModelType=ModelType.GPT_3_5_TURBO,
               model_config_dict: Optional[Dict[str, Any]]) -> ModelBackend:
```

Reply:
Left with no defaults